lib/pengine/utils.c

```diff
@@ -331,7 +328,7 @@ pe__show_node_weights_as(const char *file, const char *function, int line,
             pe_resource_t *child = (pe_resource_t *) gIter->data;
 
             pe__show_node_weights_as(file, function, line, to_log, child,
-                                     comment, nodes);
+                                     comment, child->allowed_nodes);
```

Comment:
This is an unexpected confirmation that this was a good idea ... it was totally not obvious that nodes was ignored here before